pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -1932,3 +1912,36 @@ func (service *SecurityPolicyService) BuildNetworkPolicyAllowPolicyID(uid string
 func (service *SecurityPolicyService) BuildNetworkPolicyIsolationPolicyID(uid string) string {
 	return strings.Join([]string{uid, common.RuleActionDrop}, common.ConnectorUnderline)
 }
+
+type namedPortInfo struct {
```

Comment:
it looks like this struct will be used for constructing both normal ports and named ports.
For example, 
https://github.com/vmware-tanzu/nsx-operator/pull/796/files#diff-dbfdb472df5ef985bb6da535ffdc2bf8220b8004b4e6be1cba9e81113a790344R74
where we used it for normal ports,
how about we rename it as 
`type portInfo`
this generic struct could be for both cases,  be less confuse for users.